src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,14 @@
-// your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+import React from "react";
+import Todo from "./Todo";
+function TodoList(props) {
+  return (
+    <ul>
+      {props.tasks.map((taskFromMap, index) => (
+        <Todo key={index} task={taskFromMap} />
+      ))}
```

Comment:
this works great here, but keep in mind that with this project (as compared to the one from earlier this week) the mapped-over array  objects each already have a unique property that can be used here instead of using the optional `index` parameter for .map's callback function: the `id` property each todo item has